pkg/apis/hco/v1beta1/hyperconverged_types.go

```diff
@@ -40,6 +40,8 @@ type HyperConvergedSpec struct {
 	// +optional
 	Workloads HyperConvergedConfig `json:"workloads,omitempty"`
 
+	CertConfig *HyperConvergedCertConfig `json:"certConfig,omitempty"`
+
```

Comment:
Can you please add some self-explaining comments for the OpenAPI documentation?
It will be automatically consumed by `oc explain` and by CR creation/edit form on the UI.

Reply:
Done